content/contributing/documentation/content_guidelines.rst

```diff
@@ -154,6 +155,28 @@ are also displayed in a dynamic *navigation bar* on the side.
    - :ref:`RST cheat sheet: headings <contributing/headings>`
    - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`
 
+.. _contributing/organizing-documentation:
+
+Organizing the documentation
+============================
+
+When writing documentation about a given topic, try to keep related pages organized.
+
+For most topics, a single page will do the job. Place it in the appropriate section of the
+documentation (e.g., content related to the CRM app go under :menuselection:`Applications
+-> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
+guidelines.
```

Comment:
- avoid the future tense. "a single page does the job" or "should do the job"
- 
```suggestion
For most topics, a single page should do the job. Place it in the appropriate section of the
documentation (e.g., content related to the CRM app go under :menuselection:`Applications
-> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
guidelines.
```

Reply:
I thought about this but didn't realize that "will do" was also valid in the present form.